### PR TITLE
Improve documentation slightly

### PR DESCRIPTION
**Pull request by oyvindln**
As the documentation seems to be a copy from nnedi3, it was incorrect in places, compiling in particular.

I thought I'd suggest some updates to the documentation so other people finding the library will have an easier time using it (e.g see #6).

I don't know if there are any API changes as I didn't use the library directly.

### CODE REVIEW

File: readme.rst [hunk 1]
```diff
@@ -190,19 +191,30 @@ Parameters:
         Default: False.
 
 
-Compilation
-===========
+Compilation(Linux)
```
Comment (sekrit-twc) [inline readme.rst]: space

**Comment by sekrit-twc** [comment]
You must agree to GPL2.

File: readme.rst [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Description
 ===========
 
+znedi3 is a CPU-optimized version of nneedi
```
Comment (sekrit-twc) [inline readme.rst]: spelling

Comment (sekrit-twc) [reply readme.rst]: also, period

**Comment by oyvindln** [comment]
Updated with spelling fixes.

I agree to license my contribution under the GPLv2.

**Pull request merged**